sample-fragment-deeplink/src/main/kotlin/com/github/octaone/alcubierre/sample/fragment/SampleFragment.kt

```diff
@@ -20,23 +23,49 @@ class SampleFragment : Fragment(R.layout.fmt_0) {
     private val navDrive: NavDrive get() = SampleApplication.from(requireContext()).navDrive
     private val screen: SampleScreen by screenData()
 
+    private lateinit var binding: Fmt0Binding
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        val binding = Fmt0Binding.bind(view)
+        binding = Fmt0Binding.bind(view)
 
-        binding.text.text = screen.someId.toString()
-        binding.textState.text = navDrive.state.toStackString()
+        with(binding) {
+            text.text = screen.someId.toString()
+            showStackState()
 
-        binding.btnBack.setOnClickListener {
-            navDrive.back()
-        }
+            btnBack.setOnClickListener {
+                navDrive.back()
+            }
 
-        binding.btnForward.setOnClickListener {
-            navDrive.forward("myapp://sample?id=${Counter.increment()}")
-        }
-        binding.btnNewRoot.setOnClickListener {
-            navDrive.replaceRoot(SampleScreen(Counter.increment()))
+            btnShowDialog.setOnClickListener {
+                navDrive.forward("myapp://dialog")
```

Comment:
This function can forward to the screen and also show the dialog, which are two actions different in behavior. Perhaps we should rename it